manifest.json

```diff
@@ -1,7 +1,6 @@
 {
   "name": "search-result",
   "vendor": "vtex",
-  "version": "3.56.1",
```

Comment:
Oh, sorry, I suggested that you remove this, but you just have to keep it as it was before

Reply:
My bad, updated!